src/frontend/qt_sdl/EmuSettingsDialog.cpp

```diff
@@ -24,6 +24,7 @@
 #include "Platform.h"
 #include "Config.h"
 #include "PlatformConfig.h"
+#include "../../RTC.h"
```

Comment:
the src folder is part of the include path so you don't need to specify the exact path